pages/docs/deployment/v3.mdx

```diff
@@ -1,87 +0,0 @@
----
```

Comment:
Is there an easy way to redirect all traffic to the /v3 path to the self-host-v3 one? @marcklingen

Reply:
https://github.com/langfuse/langfuse-docs/pull/948/commits/e0b61502d22fd1ad7c8ae795b43f703ba288c3c4